### ISSUE TITLE
complier not updating.

### ISSUE THREAD

**Issue by Memnon-ofol** [juanfranblanco/vscode-solidity#409]
Ive tried updating my compiler through various methods including changing it globally with no result. Any help would be appreciated.

**Comment by juanfranblanco** [comment]
Have you tried right click on a contract, and set global remote compiler? Also check that your workspace is not set as this will override the global.

**Comment by Memnon-ofol** [comment]
Yes, I have. How do I make sure my workspace doesn't override it?
